### PR TITLE
1.0 changelog!

### PR DESCRIPTION
**Pull request by haydentherapper**
Signed-off-by: Hayden Blauzvern <hblauzvern@google.com>

### CODE REVIEW

**Comment by haydentherapper** [comment]
Moved to draft, updating README

**Comment by haydentherapper** [comment]
This is ready for review, but there's a bug in verifying the certificate chain using TUF, I'm chatting with Asra about it now.

**Review by cpanato — DISMISSED** [review]

**Comment by haydentherapper** [comment]
Fixed! Ready to be merged now

**Review by priyawadhwa — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,36 +6,42 @@
...
+Fulcio is a free-to-use certificate authority for issuing code signing certificates
+for an OpenID Connect (OIDC) identity, such as email address.
 
-fulcio only signs short-lived certificates that are valid for under 20 minutes.
+Fulcio only issues short-lived certificates that are valid for 10 minutes.
 
 ## Status
 
-Fulcio is a *work in progress*!
+Fulcio is in General Availability, offering a 99.5 Availability SLO.
```
Comment (bobcallaway) [inline README.md:18]: nit: fulcio.sigstore.dev is the only thing we offer an SLO on, we might want to try to be clearer around the project and code (can be run by anyone) vs the public good instance the community runs

File: README.md [hunk 2]
```diff
@@ -49,28 +55,44 @@ KsXF+jAKBggqhkjOPQQDAwNpADBmAjEAj1nHeXZp+13NWBNa+EDsDP8G1WWg1tCM
...
+mygUY7Ii2zbdCdliiow=
+-----END CERTIFICATE-----
+```
 
 ## API
 
-The API is defined [here](./pkg/api/client.go).
-
-## Transparency
+The API is defined [here](./fulcio.proto). The API can be accessed
```
Comment (bobcallaway) [inline README.md:78]: add a link to sigstore.dev/swagger for http?

File: README.md (hunk 1 above)
Comment (bobcallaway) [reply README.md:18]: we should also explicitly acknowledge that we intend to adhere to https://semver.org/ rules around API stability

**Pull request merged**